var/spack/repos/builtin/packages/gettext/package.py

```diff
@@ -86,7 +86,6 @@ def configure_args(self):
         if '+libxml2' in spec:
             config_args.append('--with-libxml2-prefix={0}'.format(
                 spec['libxml2'].prefix))
-        else:
```

Comment:
It seems to me that `gettext` may vendor those libraries if you use the `--with-included-*` configure options:
```console
$ spack stage gettext
$spack cd gettext
$ tree -d
...
│   ├── gnulib-lib
│   │   ├── glib
│   │   ├── glthread
│   │   ├── libcroco
│   │   ├── libxml <---
│   │   ├── uniconv
│   │   ├── unictype
│   │   ├── unilbrk
│   │   ├── uniname
│   │   ├── unistr <--
│   │   └── uniwidth
...
```
Furthermore from the current configure I see:
```
checking absolute name of <libxml2/libxml/xmlexports.h>... ///home/mculpo/PycharmProjects/spack/opt/spack/linux-ubuntu14.04-x86_64/gcc-4.8/libxml2-2.9.4-een6xtvzvsfnwyhludevvuf4t6m3zxq3/include/libxml2/libxml/xmlexports.h
```
which seems to confirm the current recipe is working correctly.